allauth/socialaccount/providers/openid_connect/provider.py

```diff
@@ -12,7 +12,9 @@
 class OpenIDConnectProviderAccount(ProviderAccount):
     def to_str(self):
         dflt = super(OpenIDConnectProviderAccount, self).to_str()
-        return self.account.extra_data.get("name", dflt)
+        user_id = self.account.extra_data.get("user_id")
```

Comment:
Why prefer `user_id` -- I don't expect these to be generally recognizable for OIDC providers?